core/src/main/scala/org/http4s/multipart/MultipartParser.scala

```diff
@@ -19,7 +19,10 @@ object MultipartParser {
 
   private final case class Out[+A](a: A, tail: Option[ByteVector] = None)
 
-  def parse(boundary: Boundary): Writer1[Headers, ByteVector, ByteVector] = {
+  def parse(boundary: Boundary): Writer1[Headers, ByteVector, ByteVector] =
+    parse(boundary, 10 * 1024)
+
+  def parse(boundary: Boundary, headerLimit: Long): Writer1[Headers, ByteVector, ByteVector] = {
```

Comment:
Binary compatibility with 0.15.  I think we should collapse to a default value in 0.16.